docs/source/user/interface.rst

```diff
@@ -161,6 +161,17 @@ Toggle Simple Interface mode using the View menu:
 When you leave Simple Interface mode, the original layout of the main
 area is restored.
 
+Searching
+---------
+
+JupyterLab has an intelligent built-in search plugin for finding text within a
+notebook or other document, which uses the `Ctrl-F` shortcut by default.
```

Comment:
```suggestion
JupyterLab has a built-in search plugin enabling finding text within partially rendered
notebooks or other documents, which uses the `Ctrl-F` shortcut by default.
```

Reply:
I merged this, but changed the wording to `special` instead of intelligent as I think it better communicates the purpose and function of the plugin. I omitted the partial rendering text as I think it hurts plain-language goals/readability by adding jargon and complexity/verbosity. Related info is still included later down in the text.